src/coreclr/pal/tests/palsuite/CMakeLists.txt

```diff
@@ -322,24 +322,6 @@ add_executable_clr(paltests
   c_runtime/vsprintf/test7/test7.cpp
   c_runtime/vsprintf/test8/test8.cpp
   c_runtime/vsprintf/test9/test9.cpp
-  c_runtime/vswprintf/test1/test1.cpp
```

Comment:
These are also referenced in `compilableTests.txt` `paltestlist.txt` and `paltestlist_to_be_reviewed.txt`.

Overall in PR branch, it looks like these files needs reviewing: 
```sh
$ git grep -l snwprintf_s 
src/coreclr/pal/tests/palsuite/compilableTests.txt
src/coreclr/pal/tests/palsuite/paltestlist.txt
src/coreclr/pal/tests/palsuite/paltestlist_to_be_reviewed.txt
src/coreclr/vm/clrtocomcall.cpp
src/coreclr/vm/commtmemberinfomap.cpp
src/coreclr/vm/dispatchinfo.cpp
src/coreclr/vm/dwbucketmanager.hpp
src/coreclr/vm/dwreport.cpp
src/native/corehost/hostmisc/pal.h
src/tests/Interop/COM/NativeServer/DispatchTesting.h
```

Reply:
I'll update the txt files. The others are all conditioned to build under Windows, which is why this is okay.